src/main/java/ua/gorshkov/hw17/Person.java

```diff
@@ -0,0 +1,62 @@
+package ua.gorshkov.hw17;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+
+import java.util.ArrayList;
+
+@Entity
+public class Person {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String surname;
+
+    @Override
+    public String toString() {
+        return "Person{" +
+                "id=" + id +
+                ", name='" + name + '\'' +
+                ", surname='" + surname + '\'' + "};";
+    }
+
+    @OneToMany(mappedBy = "person", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    private List<PersonOrder> orderList = new ArrayList<>();
```

Comment:
Minor note: When we have 1-M relationship, collection of object should be with LAZY fetch type